app/src/pages/inside/common/defectTypeTooltip/defectTypeTooltip.jsx

```diff
@@ -76,25 +91,12 @@ export class DefectTypeTooltip extends Component {
              * Don't display defect sub types with zero defect's amount,
              * except system types (i.e. with index=0)
              */
-            Object.keys(data)
-              .map((key) => defectConfig.find((item) => item.locator === key))
-              .filter((item) => item)
-              .sort((a, b) => a.id - b.id)
-              .filter(({ locator }, i) => i === 0 || data[locator] > 0)
-              .map(({ locator, color, longName }) => (
-                <DefectLink
-                  key={locator}
-                  itemId={itemId}
-                  defects={[locator]}
-                  className={cx('item')}
-                >
-                  <div className={cx('name')}>
-                    <div className={cx('circle')} style={{ backgroundColor: color }} />
-                    {longName}
-                  </div>
-                  <span className={cx('value')}>{data[locator]}</span>
-                </DefectLink>
-              ))}
+            Object.keys(data).length > 2 && (
```

Comment:
According to the defect's description 
> Defect type's statistic shouldn't contain total info if there are no defect subtypes on the project

<img width="269" alt="image" src="https://user-images.githubusercontent.com/3634380/57545297-8eb92b80-7362-11e9-905e-d87171ab38f2.png">

You should leave `Product Bug` not `Total Product Bugs`

Reply:
Fixed.